autoload/neomake.vim

```diff
@@ -531,6 +536,7 @@ function! s:Make(options) abort
         let s:make_id += 1
         let s:make_options[s:make_id] = {
                     \ 'cwd': getcwd(),
+                    \ 'verbosity': get(g:, 'neomake_verbose', 1) + &verbose,
```

Comment:
I see.. :)
But then `:verb Neomake` is just using the default verbosity (1), which is what I wanted to avoid, since `:verbose` should make things more verbose after all.